README.md

```diff
@@ -28,18 +28,13 @@ npm install jasminewd2
 Usage
 -----
 
-Assumes selenium-webdriver as a peer dependency.
-
 ```js
 // In your setup.
 var JasmineRunner = require('jasmine');
 var jrunner = new JasmineRunner();
-require('jasminewd2');
+var webdriver = require('selenium-webdriver');
 
-global.driver = new webdriver.Builder().
-    usingServer('http://localhost:4444/wd/hub').
-    withCapabilities({browserName: 'chrome'}).
-    build();
+require('jasminewd2').init(webdriver.promise.controlFlow(), webdriver);
 
```

Comment:
We should probably still show this webdriver step, since we use the global `driver` below.